.pre-commit-config.yaml

```diff
@@ -40,7 +40,7 @@ repos:
       - id: codespell
         exclude: >
           (?x)^(
-              .*\.json|ignore.txt
+              .*\.json|ignore.txt|.*yaml
```

Comment:
not `.*\.yaml` because we want `_default_template_yaml` and other stubs to not have codespell either